tests/rules/test_fix_file.py

```diff
@@ -14,47 +14,47 @@
 a.c:3:1: error: expected expression before '}' token
  }
   ^
-"""),
+"""),  # noqa
 
 ('clang a.c', 'a.c', 3, 1, '',
 """
 a.c:3:1: error: expected expression
 }
 ^
-"""),
+"""),  # noqa
```

Comment:
How about this?

```python
tests = (
    ('gcc a.c', 'a.c', 3, 1, '',
     """
a.c: In function 'main':
a.c:3:1: error: expected expression before '}' token
 }
  ^
"""),

    ('clang a.c', 'a.c', 3, 1, '',
     """
a.c:3:1: error: expected expression
}
^
"""),
```

I, personally, prefer this over `# noqa` in every single line. What do you think?